sync/awsdatabricks.py

```diff
@@ -99,11 +138,37 @@ def create_prediction_for_run(
     if run["state"].get("result_state") != "SUCCESS":
         return Response(error=DatabricksError(message="Run did not successfully complete"))
 
+    run_end_time = run.get("end_time")
     cluster_response = _get_run_cluster(run["tasks"])
     if cluster := cluster_response.result:
-        eventlog_response = _get_eventlog(run["tasks"][0], cluster)
+        # Making these calls prior to fetching the event log allows Databricks a little extra time to finish
+        #  uploading all the event log data before we start checking for it
+        cluster_id = cluster["cluster_id"]
+
+        cluster_events = get_default_client().get_cluster_events(cluster_id)
+
+        aws_region_name = get_databricks_config().aws_region_name
+        ec2 = boto.client("ec2", region_name=aws_region_name)
+        instances = ec2.describe_instances(
+            Filters=[
+                {"Name": "tag:Vendor", "Values": ["Databricks"]},
+                {"Name": "tag:ClusterId", "Values": [cluster_id]},
+                # {'Name': 'tag:JobId', 'Values': []}
```

Comment:
I was considering adding this in the call, but it's not strictly necessary. Looking at it in the light of day, I'm also not 100% sure if this would play nicely with shared compute clusters vs. single-job ones. Perhaps best to leave it out